package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-swipeable-views": "^0.14.0"
+  }
+}
```

Comment:
Is this really needed at the root level?

Reply:
I didn't add this myself so I do not know if it is needed at root level or not

Reply:
@benfaustin95 Do you know why this is showing up now? I believe you were using the swipeable-views at one point

Reply:
The package is still being used at the client level, but shouldn't be at the root level. It looks like the package.json is being added with this PR is it possible the package was installed at the wrong level which generated the the file?

Reply:
Ok, in that case, the file should probably just be deleted from the root directory.